src/dune_init.ml

```diff
@@ -20,7 +20,8 @@ module Kind = struct
   let commands =
     [ "exe", Executable
     ; "lib", Library
-    ; "test", Test ]
+    ; "test", Test
```

Comment:
This commit is missing the sign off.

Reply:
Yes indeed. I was planning to squash into your commit anyways.